applications/conversations/controllers/class.messagescontroller.php

```diff
@@ -368,6 +368,14 @@ public function index($conversationID = false, $offset = -1, $limit = '') {
 
         $this->Conversation = $this->ConversationModel->getID($conversationID);
         $this->Conversation->Participants = $this->ConversationModel->getRecipients($conversationID);
+
+        //Verify if participant email should be visible
```

Comment:
Missing before and period at the end.
`// Verify if participant email should be visible.`